stdlib/REPL/test/TerminalMenus/runtests.jl

```diff
@@ -4,12 +4,9 @@ import REPL
 using REPL.TerminalMenus
 using Test
 
-TerminalMenus.config(supress_output=true)
-
-function simulateInput(expectedResult, menu::TerminalMenus.AbstractMenu, keys...)
-    # If we cannot write to the buffer, skip the test
-    !(:buffer in fieldnames(typeof(stdin))) && return true
```

Comment:
Could you say a word about this line and why this was dangerous? I don't see in which case it would have been triggered...

Reply:
It's not guaranteed if someone uses `redirect_stdin`. Looking at just the line itself, you have to dig ("what type of object is `stdin`? Where was this called from and did anyone redirect it?") just to know whether it was actually tested or not.

To me, this seems too misleading. I could change this to emit a warning, if you think that's better.

Reply:
Ok, no I agree it's better to remove it and if an error happens in a certain testing context, it seems best to handle the error at call-site (e.g. not calling this test, but explicitly).